app/src/main/java/com/example/musictoned/util/LocalStorage.kt

```diff
@@ -14,7 +14,7 @@ object LocalStorage {
     private lateinit var app: MainActivity
     fun setApplication(mainActivity: MainActivity) {
         app = mainActivity
-        //writeWorkouts(arrayListOf())
```

Comment:
Any reason for this change?

Reply:
My bad, that was for debugging and just forgot to comment it out